rootfs/scheduler/__init__.py

```diff
@@ -324,8 +324,12 @@ def create(self, name, image, command, **kwargs):
             self._create_namespace(app_name)
             self._create_minio_secret(app_name)
             self._create_service(name, app_name, app_type, data, image=image)
-            self._create_rc(name, image, command, **kwargs)
 
+            # Create RC with 0 pods and instead use scale to get polling
+            num = kwargs.pop('num')
+            kwargs['num'] = 0
+            self._create_rc(name, image, command, **kwargs)
+            self._scale_rc(name, app_name, num)
```

Comment:
yep! @helgi mentioned that in the standup. See https://github.com/helgi/workflow/blob/start_app/rootfs/scheduler/__init__.py#L585-L598 :)